src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -42,7 +42,7 @@
     },
     "conflict": {
         "symfony/config": "<2.8",
-        "symfony/dependency-injection": "<3.3",
```

Comment:
the changes on this file should be reverted. You should add the 3.4+ constraints in the FrameworkBundle composer.json for the http-kernel requirement instead (see broken tests).